### PR TITLE
Added info modal after trying to login as former member

### PR DESCRIPTION
**Pull request by Tali32**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/api/authorizeApi.ts [hunk 1]
```diff
@@ -16,8 +17,12 @@ export default class AuthorizeApi {
         }
       })
       .catch((error) => {
-        if (error.response.status === 400) {
-          notificationLogic("error", error.response.data.value);
+        if (error.response.data.value == "User-FormerMember") {
+          showUserIsFormerMemberModal();
+        } else {
```
Comment (ostappppppp) [inline src/api/authorizeApi.ts:22]: For what reason you need here this `else` statement?

File: src/api/authorizeApi.ts [hunk 2]
```diff
@@ -113,8 +118,12 @@ export default class AuthorizeApi {
...
```
Comment (ostappppppp) [inline src/api/authorizeApi.ts:123]: same here

File: src/api/authorizeApi.ts (hunk 1 above)
Comment (Tali32) [reply src/api/authorizeApi.ts:22]: Because without it there will be additional unnecessary notification

File: src/api/authorizeApi.ts (hunk 2 above)
Comment (Tali32) [reply src/api/authorizeApi.ts:123]: The same

**Review by nicoolle — APPROVED** [review]

File: src/api/authorizeApi.ts (hunk 1 above)
Comment (Tali32) [reply src/api/authorizeApi.ts]: Done!

File: src/api/authorizeApi.ts (hunk 2 above)
Comment (Tali32) [reply src/api/authorizeApi.ts]: Done!

**Review by OleksandrBilan — APPROVED** [review]

**Review by christykobyletska — APPROVED** [review]

**Review by ostappppppp — APPROVED** [review]

**Pull request merged**